Hardware/Hardware/CardReader/CardReaderService.swift

```diff
@@ -45,14 +45,13 @@ public protocol CardReaderService {
 
     /// Creates a PaymentIntent
     /// - Parameter parameters: the intent's parameters
-    func createPaymentIntent(_ parameters: PaymentIntentParameters) -> Future <PaymentIntent, Error>
+    func createPaymentIntent(_ parameters: PaymentIntentParameters) -> Future <Void, Error>
```

Comment:
I love this so much! The PI is mostly meaningless to the rest of the app. Keeping it in CardReaderService instead of passing it around, accepting it again, feels a lot safer.